api/v1alpha1/package_types.go

```diff
@@ -39,13 +41,17 @@ type PackageSpec struct {
 
 // PackageStatus defines the observed state of Package
 type PackageStatus struct {
-	Conditions     []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
-	OwnedResources []OwnedResourceRef `json:"ownedResources,omitempty"`
+	Version           string             `json:"version,omitempty"`
```

Comment:
As Helm uses chart and application versions we should also always display both versions and might display the app version more prominent than the chart version.

Reply:
This is the version of the Glasskube package. It doesn't have to be neither the chart nor app version, though it *can* be either